packages/patternfly-4/react-core/src/components/Radio/examples/DisabledRadio.js

```diff
@@ -7,8 +7,21 @@ class DisabledRadio extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <Radio aria-label="disabled checked radio example" defaultChecked isDisabled />{' '}
-        <Radio aria-label="disabled radio example" isDisabled />
+        <Radio
+          label="disabled checked radio example"
```

Comment:
Nit: Could you capitalize the first letter of the label like with the `Controlled` example.